projects/rocprofiler-systems/source/lib/core/config.cpp

```diff
@@ -2144,11 +2159,16 @@ get_perfetto_backend()
 std::string
 get_perfetto_output_filename()
 {
-    static auto _v       = get_config()->find("ROCPROFSYS_PERFETTO_FILE");
-    auto        _val     = static_cast<tim::tsettings<std::string>&>(*_v->second).get();
-    auto        _pos_dir = _val.find_last_of('/');
-    auto        _dir     = std::string{};
-    auto        _ext     = std::string{ "proto" };
+    static auto _v   = get_config()->find("ROCPROFSYS_PERFETTO_FILE");
+    auto        _val = static_cast<tim::tsettings<std::string>&>(*_v->second).get();
+
+    ROCPROFSYS_BASIC_VERBOSE_F(
+        2, "[get_perfetto_output_filename] Initial ROCPROFSYS_PERFETTO_FILE='%s'\n",
+        _val.c_str());
```

Comment:
Not sure if you intended to keep in all of these logs. However, if you do, you don't need to include the function name. The "_F" macros should already prefix it. 

For example,
``` 
[rocprof-sys][2336120][get_perfetto_output_filename][get_perfetto_output_filename] Initial ROCPROFSYS_PERFETTO_FILE='perfetto-trace.proto'
```

```suggestion
    ROCPROFSYS_BASIC_VERBOSE_F(
        2, "Initial ROCPROFSYS_PERFETTO_FILE='%s'\n",
        _val.c_str());
```

Reply:
Missed during cleanup. It was my debug log for testing.